src/components/NavPrimary/NavPrimary.tsx

```diff
@@ -287,16 +292,21 @@ const NavPrimary = ({
 
   return (
     <header ref={navRef} className={'bsds-nav-header' + (isConstrained ? ' is-constrained' : '')}>
-      {!!utilityItems && <NavUtility utilityItems={utilityItems} ariaLabel={texts?.utilityNavAriaLabel} />}
+      {!!utilityItems && (
+        <NavUtility utilityItems={utilityItems} ariaLabel={texts?.utilityNavAriaLabel} logoSecondary={logoSecondary} />
+      )}
       <nav className="bsds-nav-primary" aria-label={texts?.primaryNavAriaLabel || 'primary'}>
         <div className="bsds-nav-bar">
-          {logo.to || logo.href ? (
-            <Link to={logo.to} href={logo.href} className="bsds-nav-link-logo" isNavLink>
+          <div className="bsds-nav-logo">
+            {logo.to || logo.href ? (
+              <Link to={logo.to} href={logo.href} isNavLink>
+                <img src={logo.src} alt={logo.alt} />
+              </Link>
+            ) : (
               <img src={logo.src} alt={logo.alt} />
-            </Link>
-          ) : (
-            <img className="bsds-nav-link-logo" src={logo.src} alt={logo.alt} />
-          )}
+            )}
+          </div>
```

Comment:
`bsds-nav-link-logo` was previously the classname for the <Link>, but not the image inside it, and the un-linked image. This change from `bsds-nav-link-logo` to `bsds-nav-logo` gives the link and image (linked or unlinked) a common parent and style.